### ISSUE TITLE
A little confused by continuing training

### ISSUE THREAD

**Issue by CAT1210** [AlexeyAB/darknet#8188]
I have been reading up on continuing training on a custom dataset. I have a dataset that is about 50,000 images and training on these images is great, but takes quite a long while. So, I want to interrupt the training when the mAP looks good and the loss is low. However, my dataset is constantly growing for false alerts and more refined images and I wish to continue training rather than start all over.

When I collect new images for my dataset and annotate them, and I want to start training again, can I just train on the new images in my dataset by starting with my last weights file or do I have to add the new images to the old dataset and train from last weights with all images? I know I can clear flags if I were to adjust the number of images I'm training on and I figure that perhaps training from last weights is like starting training from the downloaded weight file at the beginning but don't know if that is really how it works?

**Comment by haviduck** [comment]
add new ones, populate train.txt and add the last weight to the cmd :) i do it all the time with pseudolabeling

**Comment by CAT1210** [comment]
It sounds like you are saying you add the new files and annotation files to your original images and annotation files and add them to the train.txt files, is that correct? If so, what do you do when you add enough files that you pass the max batches in the config file or do you push that up far enough to not have a problem with that? Or do you adjust max batches as well and clear flags and continue with last weights?

**Comment by haviduck** [comment]
> It sounds like you are saying you add the new files and annotation files to your original images and annotation files and add them to the train.txt files, is that correct? If so, what do you do when you add enough files that you pass the max batches in the config file or do you push that up far enough to not have a problem with that? Or do you adjust max batches as well and clear flags and continue with last weights?

yeah basic pseudolabelling. start manually tagging a small batch, train it for a little bit then i use that on a larger set, rinse and repeat until ive gotten through the dataset i need. continuing training is pretty straight forward, and you can use the basemodel as a template with a different cfg. dont have to continue on the exact same naming convention. 

darknet.exe detector train data/obj.data yolo-obj.cfg backup\yolo-obj_2000.weights
